app/src/main/res/layout/fragment_qrcode.xml

```diff
@@ -20,16 +20,17 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingBottom="@dimen/activity_vertical_margin"
-    android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
-    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="0dp"
+    android:paddingLeft="0dp"
+    android:paddingRight="0dp"
+    android:paddingTop="0dp"
     tools:context=".ui.activities.WifiNetworkActivity$QrCodeFragment">
 
     <ImageView
         android:id="@+id/qr_code"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```

Comment:
I wouldn't set the layout width and height to `match_parent`; on a 10" tablet it looks way to big and is a bit blurry because the PNG is scaled up. We can make it fill smaller screens but keep it at 5-10 cm max on bigger ones.